coforma/swift-tech-challenge
# Add card icons and 'back to top' button

## Summary

We need icons for browse cards and a button on the browse page to take users back to the top. This PR adds both.
Closes #77 
Fixes #56 (issue)

## Type of change

## How To Test
1. Pull and run app
2. Observe the pleasant way the back-to-top button appears and disappears, and the gentleness of its scroll. 
3. Bask in the beauty of the icons on the cards.

# Checklist:
